Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "AutoHashEquals"
+uuid = "97026771-8bcd-44fb-b7eb-f644370d63d3"
+authors = []
```

Comment:
Also, @iamed2: I left this blank, but please feel free to fill it out; you know the history of this package better than I do.

Reply:
I would just put Andrew there, since all other contributions have just been small upgrades for new Julia versions.

https://github.com/andrewcooke/AutoHashEquals.jl/graphs/contributors

Reply:
Done. Thanks!